account-abstraction/gas_manager_admin.yaml

```diff
@@ -22,7 +22,7 @@ paths:
                   description: String ID of the app
                   default: '6d834x9k1yh4dx6z'
                 rules:
-                  $ref: './schemas.yaml#/components/schemas/Rules'
+                  $ref: './account-abstraction/schemas.yaml#/components/schemas/Rules' # path for using github actions
```

Comment:
in the comments, can u add the path if ppl wanted to do a manual update? e.g './schemas.yaml#/components/schemas/Rules'